docs/advanced.md

```diff
@@ -421,6 +421,41 @@ class MyCustomAuth(httpx.Auth):
         ...
 ```
 
+Similarly, if you are implementing a scheme that requires access to the response body, then use the `requires_response_body` property.   You will then be able to access response body properties and methods such as `response.text`, `response.json()`, `response.iter_lines()`, etc.
```

Comment:
I'd suggest "methods such as `response.content`, response.text`, and `response.json()``"

The `response.iter_lines()` is kind of valid here, but really it's an API that's only there in order to support *streaming* responses.